beetsplug/mbcollection.py

```diff
@@ -28,8 +28,6 @@
 
 SUBMISSION_CHUNK_SIZE = 200
 UUID_REGEX = r'^[a-f0-9]{8}(-[a-f0-9]{4}){3}-[a-f0-9]{12}$'
-AUTH_ERRORS = (TokenRequestDenied, TokenMissing, VerifierMissing)
-
```

Comment:
I thought about it, but I am using rauth lib and these are from requests_oauthlib.